doc-content/kogito-docs/src/main/asciidoc/creating-running/con_kogito-automation.adoc

```diff
@@ -53,4 +53,4 @@ Another focus of {PRODUCT} is optimum developer experience. You can use much or
 .Building a {PRODUCT} process service in VSCode
 image::kogito/kogito-run-quarkus-devmode-demo.gif[Demo of {PRODUCT} BPMN2 extension in VSCode]
 
-To deploy your services into the cloud, you can use the {PRODUCT} Operator, which guides you through every step. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to where your application code lives in Git, and the Operator can check it out, build it (including native compilation, if applicable), and deploy the resulting services. {PRODUCT} also offers a Command Line Interface (CLI) to simplify some of these deployment tasks.
+To deploy your services to the cloud, you can use the {PRODUCT} Operator, which guides you through the deployment process. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to the Git repository that contains your application, and the operator can pull the project, build it (including native compilation, if applicable), and deploy the resulting services. {PRODUCT} also offers a Command Line Interface (CLI) to simplify some of these deployment tasks.
```

Comment:
On "For example,", that's not a 100% true and might confuse users.
The operator don't pull the code itself, it create the BuildCOnfig object on the openshift cluster and ocp will pull  and build it.

it might be true for kubernetes, but IIRC the kogito cli will do this.

@ricardozanini can you please confirm?

Reply:
Hi!
We do not support Kubernetes for now, so won't be a problem and yes, the plan is to have the CLI to build the image locally since we don't have this kind of support on Kube. 

I think adding to the paragraph that the "Operator will create a BuildConfig with s2i strategy to pull the code from git" is too much information, we can hide some technical details at this point since it's a getting started. Another more "in depth" doc could have all of this. But I might be wrong, @sterobin wdyt?

Reply:
@spolti, agreed with @ricardozanini, too much info I think. How about "... and the operator can pull the project **configurations**, build **the project** (including ...." Would that be better?

Reply:
the point is, for now, operator does not pull or build anything.

Reply:
@spolti, from your comment above:

>  it create the BuildCOnfig object on the openshift cluster and ocp will pull **and build it**

So I'm unclear.

Reply:
Since the operator is handling and managing the OpenShift resources that actually do the build, I do not see any harm saying that the Operator will pull and build the service.

Reply:
Well, I agree with @spolti, as it is technically not correct ... Can't we say something like this:

The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] to automate and manages many of the deployment steps for you. For example, you can provide the Operator a link to the Git repository that contains your application, which enables the Operator to automatically configure the components necessary to build your project from source (including native compilation, if applicable), and deploy the resulting services. {PRODUCT} also offers a Command Line Interface (CLI) to simplify some of these deployment tasks.

Reply:
Duncan, ack.

@DuncanDoyle, @spolti, @ricardozanini, agreed with Duncan's explanation, with some slight tweaks as follows. The changes from the original text are in **bold**. Work for you guys?

>The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] **to automate and manage** many of the deployment steps for you. For example, **when you** provide the Operator a link to the Git repository that contains your application, **the operator can automatically configure the components required to build your project from source** (including native compilation, if applicable) and deploy the resulting services. {PRODUCT} also offers a Command Line Interface (CLI) to simplify some of these deployment tasks.

Reply:
+1 from Duncan suggestion.

@sterobin yes, here in community we could use native builds, but for product, I can't say we can't use, but we need to stay in sync with Quarkus team about this.

Reply:
@spolti, I see. For simplicity, I'll just remove that I think. And I just removed in my proposed text above.